src/equations/shallow_water_multilayer_1d.jl

```diff
@@ -396,7 +492,7 @@ end
     c_ll = sqrt(equations.gravity * sum(h_ll))
     c_rr = sqrt(equations.gravity * sum(h_rr))
 
-    return (max(abs(v_m_ll) + c_ll, abs(v_m_rr) + c_rr))
+    return (max(abs(v_m_ll), abs(v_m_rr)) + max(c_ll, c_rr))
```

Comment:
Why replace the Davis wave speed estimate here?

Reply:
I changed the wave speed estimate to be consistent with the `max_abs_speed_naive` function as implemented in Trixi.jl.